packages/calcite-components/src/components/shell/shell.e2e.ts

```diff
@@ -122,3 +122,41 @@ describe("calcite-shell", () => {
     expect(panelTop).toBeNull();
   });
 });
+
+describe("slotted overrides", () => {
```

Comment:
Outer-most describe block is not consistent with [suggested pattern](https://github.com/Esri/calcite-design-system/blob/0956bad28d2693ac2a9b207523878ef66266593f/packages/calcite-components/src/tests/commonTests/themed.ts#L28).

Reply:
whoops!